include/ibamr/IBFESurfaceMethod.h

```diff
@@ -521,13 +547,23 @@ class IBFESurfaceMethod : public IBStrategy
     const unsigned int d_num_parts;
     std::vector<IBTK::FEDataManager*> d_fe_data_managers;
     SAMRAI::hier::IntVector<NDIM> d_ghosts;
-    std::vector<libMesh::System*> d_X_systems, d_U_systems, d_U_n_systems, d_U_t_systems, d_F_systems, d_DP_systems;
-    std::vector<libMesh::PetscVector<double>*> d_X_current_vecs, d_X_new_vecs, d_X_half_vecs, d_X0_vecs, d_X_IB_ghost_vecs;
+    std::vector<libMesh::System*> d_X_systems, d_U_systems, d_U_n_systems, d_U_t_systems, d_F_systems, d_WSS_o_systems,
```

Comment:
Can you add some comments about these different systems?

@drwells should we make some data structure for packaging together `System`s and `PetscVector`s. This way of doing things is my fault, but it seems silly that if we want to include a new variable here, we repeat the name 2 or 3 times (once for the system, and then once for each of the DOF vectors that we explicitly track).